server/bound_queries.cc

```diff
@@ -625,6 +722,150 @@ struct RowHashOrderedExecutor: public BoundSelectQuery::Executor {
         return;
     }
 
+    virtual void execute_iter(std::function<bool (const NamedRowValue & output,
```

Comment:
Comment what's the difference between execute_iter (executeIterative?) and execute_bloc (executeBlock?)